.github/workflows/publich

```diff
@@ -0,0 +1,57 @@
+name: publish-postgresql-helm-chart
+
+on:
+  push:
+    branches: ['main']
```

Comment:
`master`

Reply:
I'm not sure that we want to trigger on push to `master`. This would allow accidental overwrite of a previously published version if a commit it merged without the version being bumped in the chart